gatsby-config.js

```diff
@@ -135,6 +135,17 @@ module.exports = {
                     BASE_URL +
                     (blogPostMeta.shareImage.childImageSharp.fixed.src ||
                       DEFAULT_META_IMAGE_URL_PATH);
+                  const coverImage =
+                    blogPostMeta.attribution && blogPostMeta.attribution.creator
+                      ? `
+                    <figure>
+                      <img src='${imageUrl}' alt=''>
+                      <figcaption>${blogPostMeta.attribution.creator} - ${blogPostMeta.attribution.source}</figcaption>
```

Comment:
```suggestion
                      <figcaption>Image by ${blogPostMeta.attribution.creator} - ${blogPostMeta.attribution.source}</figcaption>
```

This should clarify what it is. I'm okay with presenting the attributes here, we don't need to hide them.

Reply:
Good idea. I will also try to combine the text and the link with an `a` inside `figcaption`. And add a separtor between the caption and the preview text. Ideal solution:
```
Image by Daniel Kudela · Before starting a React based project, the same question often arises: Which framework should be used? Of course, this question cannot be answered in a general way, since each framework has its advantages and disadvantages for specific use cases. Nevertheless, you can compare the frameworks, especially the concepts behind them.
```